CONTRIBUTING.md

```diff
@@ -0,0 +1,180 @@
+$ cd Mirage-UI
+$ git remote add upstream https://github.com/ALPHAVIO/Mirage-UI
+```  
+![ForkDemo](https://user-images.githubusercontent.com/65714751/121729332-fa56bf80-cb0b-11eb-872e-5f0c68ef5d12.png)
+
+
+
+- If you have already forked the project, update your copy before working.  
+```  
+$ git remote update  
+$ git checkout <branch-name>  
+$ git rebase upstream/<branch-name>  
+```  
+## Step 2 : Branch
+**Make a branch only if you are working on a major feature otherwise skip this step.**
+Create a new branch. Use its name to identify the issue your addressing.  
+```  
+# It will create a new branch with name Branch_Name and switch to that branch
+$ git checkout -b branch_name  
+```  
+
+Our branching strategy is based on [this article](https://nvie.com/posts/a-successful-git-branching-model/) which I suggest you read.
+
++  **master** a history of releases, once merged to from develop and tagged we create a release on the play store & GitHub releases.
+
++  **develop**  the actively worked on next release of the app, what we branch off of while working on new features and what we merge into upon feature completion
+
++ **feature/** or feature/\<username\>/ any branch under this directory is an actively developed feature, feature branches culminate in a PR, are merged and deleted.
+ Typically a feature branch is off of develop and into develop but in rare scenarios if there is an issue in production a branch may be made off master to fix this issue, this type of feature branch must be merged to develop and master before being deleted.
+Branch names should be in the format **\<issue-number\>-kebab-case-title**
+
+All branches should have distinct history and should be visually easy to follow, for this reason only perform merge commits when merging code either by PR or when synchronising.
+
+If you wish to rebase you should be following the [Golden Rule](https://www.atlassian.com/git/tutorials/merging-vs-rebasing#the-golden-rule-of-rebasing) and ahere to the advice in the heading [Aside: Rebase as cleanup is awesome in the coding lifecycle](https://www.atlassian.com/git/articles/git-team-workflows-merge-or-rebase).
+
+## Step 3 : Work Remotely  
+- Now you are ready to your work to the remote repository.
+
+## Step 4 : Work on the feature/ issue.  
+- Add all the files/folders needed.  
+- After you've made changes or made your contribution to the project add changes to the branch you've just created by:  
+```  
+# To add all new files to branch Branch_Name  
+$ git add .  
+
+# To add only a few files to Branch_Name
+$ git add <file name>
+```
+## Step 5 : Lint Your Code  
+- To check for any linting errors, run command:
+```
+npm run lint-check
+```
+- To fix linting errors, run command:
+```
+npm run lint
+```
+- If some files have changed by above command, you have to repeat Step 4
+- Note : If you get error "eslint is not recognised", then you have to install eslint by using following command
+```
+npm install -g eslint 
+```
```

Comment:
replace this step 5 since our root directory does not have a package.json file.

Just add instruction with a couple links to format the code using prettier or any linter of choice (eslint preferred)